### ISSUE TITLE
Exception "Unknown op 0" with some woff fonts

### ISSUE THREAD

**Issue by alafr** [foliojs/fontkit#58]
Fontkit (used in PDFKit) throws an exception with some fonts in [this folder](https://github.com/w3c/web-platform-tests/blob/master/svg/import/woffs/):
- [Blocky.woff](https://github.com/w3c/web-platform-tests/blob/master/svg/import/woffs/Blocky.woff)
- [FreeMono.woff](https://github.com/w3c/web-platform-tests/blob/master/svg/import/woffs/FreeMono.woff)
- [PlaneOne.woff](https://github.com/w3c/web-platform-tests/blob/master/svg/import/woffs/PlaneOne.woff)
- [embeded.woff](https://github.com/w3c/web-platform-tests/blob/master/svg/import/woffs/embeded.woff)

Most are testing fonts with lots of missing glyphs but apparently that's not the issue because they work as expected if converted to TTF.

```
Error: Unknown op: 0
    at parse (http://pdfkit.org/demo/bundle.js:39197:23)
    at CFFGlyph._getPath (http://pdfkit.org/demo/bundle.js:39213:7)
    at CFFGlyph.get (http://pdfkit.org/demo/bundle.js:38136:19)
    at CFFGlyph.descriptor.get (http://pdfkit.org/demo/bundle.js:29165:21)
    at CFFGlyph._getCBox (http://pdfkit.org/demo/bundle.js:37980:18)
    at CFFGlyph.get (http://pdfkit.org/demo/bundle.js:38111:19)
    at CFFGlyph.descriptor.get (http://pdfkit.org/demo/bundle.js:29165:21)
    at CFFGlyph._getMetrics (http://pdfkit.org/demo/bundle.js:38024:22)
    at CFFGlyph.get (http://pdfkit.org/demo/bundle.js:38147:19)
    at CFFGlyph.descriptor.get (http://pdfkit.org/demo/bundle.js:29165:21)
```

**Comment by alafr** [comment]
I tried to debug this issue. I compared with opentype.js which works with these fonts.
Apparently the `stream` used in the function   [`CFFGlyph`>`_getPath`>`parse`](https://github.com/devongovett/fontkit/blob/master/src/glyph/CFFGlyph.js#L84) is not the correct one, and replacing `stream` with `cff.stream` solves the problem for the `.woff` fonts mentionned above.
But I can't make a pull request because I'm not familiar with fonts and it would probably break everything else...
